ui/app/pages/permissions-connect/permissions-connect.component.js

```diff
@@ -150,58 +159,69 @@ export default class PermissionConnect extends Component {
       permissionsRequest,
       addressLastConnectedMap,
       permissionsRequestId,
+      connectPath,
+      confirmPermissionPath,
+      page,
     } = this.props
-    const { page, selectedAccountAddress, permissionAccepted, originName } = this.state
+    const { selectedAccountAddress, permissionAccepted, originName, redirecting } = this.state
 
     return (
       <div className="permissions-connect">
-        { page !== null
+        { redirecting
```

Comment:
This should be `!redirecting`
```suggestion
        { !redirecting
```

Reply:
I got this wrong in both places in my last review it seems 🤦‍♂